en/docs/Learn/APISecurity/OpenIDConnect/obtaining-user-profile-information-with-openid-connect.md

```diff
@@ -63,28 +61,25 @@ For an online tool to decode the JWT, go to <https://jwt.io/>
 
 You can obtain user information as a payload by invoking the userinfo endpoint with the access token obtained in step 1. The format of the curl command and a sample is given below
 
--   [**Format**](#150e27cf5d164e81a3a6cd46d4365460)
--   [**Sample**](#2f76f7b5df6b4adfb6dbd617f3763925)
-
-``` java
-    curl -k -v -H "Authorization: Bearer <ACCESS_TOKEN>" https://<GATEWAY_HOSTNAME>:<PORT>/userinfo
+``` bash tab="Format"
+curl -k -v -H "Authorization: Bearer <ACCESS_TOKEN>" https://<GATEWAY_HOSTNAME>:<PORT>/userinfo
 ```
 
-``` java
-    curl -k -v -H "Authorization: Bearer 83705add-d77e-3cc8-9b6a-53d210ed3fed" https://apim.wso2.com:8243/userinfo
+``` bash tab="Example"
+curl -k -v -H "Authorization: Bearer 83705add-d77e-3cc8-9b6a-53d210ed3fed" https://apim.wso2.com:8243/userinfo
 ```
 
 The response will be a JSON payload as shown below:
 
 ``` java
-    {
-      "sub": "user1@carbon.super",
-      "organization": "WSO2",
-      "email": "user1@gmail.com"
-    }
+{
+  "sub": "user1@carbon.super",
+  "organization": "WSO2",
+  "email": "user1@gmail.com"
+}
 ```
 
 !!! note
-By default, only the username (sub) information will be available in the response. You can customize the user information returned by configuring the claims of the relevant Service Provider generated for the Application created in Store. For more information, see [Service Provider Claim Configuration](https://docs.wso2.com/display/IS530/Configuring+Claims+for+a+Service+Provider) .
+    By default, only the username (sub) information will be available in the response. You can customize the user information returned by configuring the claims of the relevant Service Provider generated for the Application created in Store. For more information, see [Service Provider Claim Configuration](https://is.docs.wso2.com/en/5.9.0/learn/configuring-claims-for-a-service-provider/) .
```

Comment:
do you customize it in the store? or is the service provider generated in the store?

Reply:
Added information to locate the relevant service provider via management console